behind-the-veil-siteroot/imports/ui/components/pages/cancel-booking/CancelBooking.jsx

```diff
@@ -5,75 +5,189 @@
  */
 
 import React, {useState} from "react";
+import {useNavigate, useParams} from "react-router-dom";
 import ServiceDetailsHeader from "../../service-details-header/ServiceDetailsHeader";
 import WhiteBackground from "../../whiteBackground/WhiteBackground";
 import PageLayout from "../../../enums/PageLayout";
 import Button from "../../button/Button";
-import {CheckIcon} from "@heroicons/react/24/outline"
+import {CheckIcon, NoSymbolIcon} from "@heroicons/react/24/outline"
 import BackButton from "../../button/BackButton";
+import {useSubscribe, useTracker} from "meteor/react-meteor-data";
+import Loader from "../../loader/Loader";
+import {getUserInfo} from "/imports/ui/components/util"
```

Comment:
This util was renamed to useUserInfo in another pr b/c it's a custom react hook. Pls update to match

Reply:
Done